### PR TITLE
Mac build

### PR DESCRIPTION
**Pull request by rwmcintosh**

### CODE REVIEW

File: src/OSPSuite.FuncParser/OSPSuite.FuncParser.csproj [hunk 1]
```diff
@@ -19,10 +19,20 @@
     <Copyright>© Open Systems Pharmacology Community 2020</Copyright>
     <Product>OSPSuite FuncParser</Product>
     <Configurations>Debug;Release;Purify_Debug</Configurations>
+    <NuspecProperties>version=$(PackageVersion)</NuspecProperties>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' ">
+  <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('Windows'))">
```
Comment (Yuri05) [inline src/OSPSuite.FuncParser/OSPSuite.FuncParser.csproj:25]: just curious: what's the advantage of `[MSBuild]::IsOSPlatform` vs. `'$(OS)' ==` ?

**Comment by Yuri05** [comment]
seems to work and all tests are running under MacOS. Very nice!

**Review by Yuri05 — APPROVED** [review]

File: src/OSPSuite.FuncParser/OSPSuite.FuncParser.csproj (hunk 1 above)
Comment (rwmcintosh) [reply src/OSPSuite.FuncParser/OSPSuite.FuncParser.csproj:25]: The `$(OS)` is limited and does not distinguish Unix from Mac

https://github.com/dotnet/msbuild/issues/2468#issuecomment-324694593

Comment (Yuri05) [reply src/OSPSuite.FuncParser/OSPSuite.FuncParser.csproj:25]: ok, thx

File: buildNix.sh
Comment (Yuri05) [inline buildNix.sh:1]: another question: what does "Nix" stand for here?

Comment (rwmcintosh) [reply buildNix.sh:1]: I'm using it as short for any Unix variant - Mac and Linux fall into that category.

Comment (Yuri05) [reply buildNix.sh:1]: That was my assumption, just didn't know that MacOS is based on Unix.
Ok, learned something new :)

**Pull request merged**